src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,7 +207,7 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().startsWith(resource.getId())) {
+            if (trigger.getId().endsWith(resource.getId())) {
```

Comment:
Perhaps, you would like to check the trigger with the tags instead the id (as commented on the IRC) ?